{{cookiecutter.project_name}}/docker/docker-compose.prod.yml

```diff
@@ -9,7 +9,7 @@
 version: "3.6"
 services:
   caddy:
-    image: "caddy:2.0.0"
+    image: "caddy:2"
```

Comment:
Let's use `image: "caddy:2.2"`

Reply:
@sobolevn , there is no caddy@2.2 as a shared tag. Only next available tags: `2.2.1`, `2`, `latest`. More details at [DockerHub](https://hub.docker.com/_/caddy)

Reply:
Packaging is hard 😒

Reply:
`2.2.1` then!